Classes/Steno.sc

```diff
@@ -411,25 +413,28 @@ Steno {
 
 
 	declareVariables { |names|
+		var bus;
 		names.do { |name|
 			name = name.asSymbol;
 			if(variables[name].isNil) {
 				"new variable as ".post;
+				bus = Bus.audio(server, numChannels);
```

Comment:
If you move this bus allocation out of the synth function, you have to take care to correctly treat it in the method:

```supercollider
rebuildSynthDefs {
		variables = (); // declaration happens in func
		encyclopedia.keysValuesDo { |key, func| SynthDef(this.prefix(key), func).add };
	}
```

the placement inside the synth func was a way to automatically restore the busses with the correct channel numbers, when e.g. the number of channels is changed on the fly.

Reply:
ok, will re-integrate